go/imageio/backend.go

```diff
@@ -1,10 +1,5 @@
-// ovirt-imageio
-// Copyright (C) 2021 Red Hat, Inc.
-//
-// This program is free software; you can redistribute it and/or modify
-// it under the terms of the GNU Lesser General Public License as
-// published by the Free Software Foundation; either version 2 of the
-// License, or (at your option) any later version.
+// SPDX-FileCopyrightText: Red Hat, Inc.
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
You replaced it with GPL!

I'm not sure if LGPL actually works with the way Go projects are built
but this commit should not change the license.

We need to add LGPL license to LICENCES/ and use the right spdx header
for this directory.